Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.10-alpine AS build
```

Comment:
```suggestion
FROM golang:1.11-alpine AS build
```

Reply:
Amended in pull #18 which is just the Dockerfile (with a bump on the golang version of the build container)